config/default.yml

```diff
@@ -4374,6 +4374,7 @@ Style/MultipleComparison:
   VersionAdded: '0.49'
   VersionChanged: '1.1'
   AllowMethodComparison: true
+  MinComparisonsCount: 2
```

Comment:
I'd name this "AllowedComparisonsCount" or "ThresholdComparisonsCount". I don't think that "Min" is the best term in the context. Perhaps the word "count" is a bit redundant.

Reply:
(obviously the default value would be different depending on the exact name chosen)

Reply:
Changed to `ComparisonsThreshold`. Let me know what you think.